src/components/ui-shell/side-nav.ts

```diff
@@ -81,6 +81,9 @@ class BXSideNav extends HostListenerMixin(LitElement) {
    */
   private _handleButtonToggle(event: CustomEvent) {
     this.expanded = event.detail.active;
+    if (this.expanded) {
+      this.querySelectorAll((this.constructor as typeof BXSideNav).selectorNavItems)[0].focus();
```

Comment:
```suggestion
      (this.querySelector((this.constructor as typeof BXSideNav).selectorNavItems) as HTMLElement).focus();
```

* `.querySelector(selector)` returns the first match, and thus equivalent to `.querySelectorAll(selector)[0]`.
* `.querySelector()` returns an `Element`, but `Element` doesn't have `.focus()`. `HTMLElement` (an inheritance of `Element`) does. (Meaning, doing so avoids TS error)

Reply:
Thanks for the insights!